tests/models/test_simplemodel.py

```diff
@@ -39,3 +42,21 @@ def test_simplemodel(testdata_3days_path):
             track_b,
             round_lon_lats=True)
     assert rarest_measurement_pair
+
+    # test MeasurementPairClassifier
+    model = MeasurementPairClassifier(
+        colocated_training_data=parse_measurements_csv(
+            test_measurements_path))
+    data = list(dummy_cruncher(test_data))
+    lrs, y_true = run_pipeline(data, model)
+
+    assert lrs == [1.0, 1.0, 1.0, 1.0]
+    assert y_true == [True, True, False, False]
+
+    # test select_colocated_pairs
+    assert len(model.colocated_training_pairs) == 419
+    assert model.colocated_training_pairs[0].is_colocated == True
+    assert model.colocated_training_pairs[212].distance == 20458.722810043124
```

Comment:
Equality check on a floating point is bound to cause issues, how much accuracy would be considered the 'right' value?

```suggestion
    assert model.colocated_training_pairs[212].distance == pytest.approx(20458.7)
```